sdk/servicebus/azure-messaging-servicebus/pom.xml

```diff
@@ -47,12 +47,12 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core-experimental</artifactId>
-      <version>1.0.0-beta.9</version> <!-- {x-version-update;unreleased_com.azure:azure-core-experimental;dependency} -->
+      <version>1.0.0-beta.8</version> <!-- {x-version-update;com.azure:azure-core-experimental;dependency} -->
```

Comment:
Is beta.8 released? I don't see it anywhere in our version_client.txt.

Reply:
Alan released along with core release on Oct 29th https://search.maven.org/artifact/com.azure/azure-core-experimental/1.0.0-beta.8/jar